docs/docusaurus/docs/reference/learn/data_quality_use_cases/dq_use_cases_lp.md

```diff
@@ -34,4 +34,6 @@ A comprehensive data quality strategy relies on a multidimensional approach to a
 
   <LinkCard topIcon label="Volume" description="Validate that record quantity falls within expected bounds." to="/reference/learn/data_quality_use_cases/volume" icon="/img/actions_icon.svg"/>
 
+  <LinkCard topIcon label="Unstructured data" description="Validate unstructured data." to="/reference/learn/data_quality_use_cases/unstructured_data" icon="/img/actions_icon.svg"/>
```

Comment:
thanks for updating the casing of this card title. I still think the [description is circular](https://github.com/great-expectations/great_expectations/pull/11380#discussion_r2359648519) and should be revised. Here's a suggestion for that

 ```suggestion
  <LinkCard topIcon label="Unstructured data" description="Validate structured metadata generated from unstructured content." to="/reference/learn/data_quality_use_cases/unstructured_data" icon="/img/actions_icon.svg"/>
```